ably/rest/auth.py

```diff
@@ -148,25 +169,29 @@ def request_token(self, key_id=None, key_value=None, query_time=None,
         )
 
         AblyException.raise_for_response(response)
+        #access_token = response.json()["token"]
+        #log.debug("Token: %s" % str(access_token))
```

Comment:
I would really prefer we either log this as level debug, or we remove it, but keeping commented out code like this is unnecessary